v1.0.0 was incorrectly bundled to npm

when installing and running `v1.0.0` the following error is thrown

```console
node:internal/modules/cjs/loader:1222
  throw err;
  ^

Error: Cannot find module './exercises/menu.json'
Require stack:
```

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
   },
   "files": [
     "index.js",
-    "examples",
+    "exercises",
     "i18n",
     "lib",
     "patches"

